src/controller.py

```diff
@@ -107,6 +107,9 @@ def run_command_and_get_its_response(self) -> dict:
         elif self.command == "emulator-read-and-confirm-mnemonic":
             emulator.read_and_confirm_mnemonic()
             return {"response": "Read and confirm mnemonic"}
+        elif self.command == "emulator-read-and-confirm-mnemonic-shamir":
+            emulator.read_and_confirm_mnemonic_shamir()
+            return {"response": "Read and confirm mnemonic SHAMIR"}
```

Comment:
`Read and confirm Shamir mnemonic`

Reply:
Also we should use past tense here shouldn't we? So `Read and confirmed`. But feel free to do that in another PR.

Reply:
Yes, good idea, will do it after this PR